bitcoin-core-dev-tech/2019-06-05-wallet-architecture.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: Wallet Architecture (2019-06-05)
+title: Wallet Architecture 
 transcript_by: Bryan Bishop
 categories: ['core-dev-tech']
 tags: ['wallet', 'bitcoin core']
+date: 2019-06-05
```

Comment:
I think it's safe to assume that Andrew Chow is the speaker here

Reply:
Noted.

Reply:
Noted.